src/main/java/roomescape/repository/ThemeRepository.java

```diff
@@ -0,0 +1,16 @@
+package roomescape.repository;
+
+import roomescape.domain.Theme;
+
+public interface ThemeRepository {
+    String FIND_THEME_BY_ID_SQL = "SELECT * FROM theme WHERE id = ?;";
+    String UPDATE_THEME_SQL = "UPDATE theme SET name = ?, desc = ?, price = ? WHERE id = ?;";
+    String DELETE_THEME_SQL = "DELETE FROM theme WHERE id = ?;";
+
+    Long createTheme(Theme theme);
+    Theme findThemeById(Long id);
```

Comment:
자바에서 null 을 좀 더 우아하게 처리할 수 있는 Optional을 사용해보면 좋을 것 같아요 :)
https://homoefficio.github.io/2019/10/03/Java-Optional-%EB%B0%94%EB%A5%B4%EA%B2%8C-%EC%93%B0%EA%B8%B0/

```suggestion
    Optional<Theme> findThemeById(Long id);
```